File: mobile-client/app/src/main/java/be/kdg/mobile_client/round/Round.java
package be.kdg.mobile_client.round;

import java.util.List;

import be.kdg.mobile_client.room.model.Card;
import be.kdg.mobile_client.room.model.Phase;
import be.kdg.mobile_client.room.model.Player;
import lombok.Data;

@Data
public class Round {
    private int id;
    private List<Card> cards;
    private List<Player> playersInRound;
    private Phase currentPhase;
    private int button;
    private boolean finished;
    private int pot;
    private int bigBlind;
    private int smallBlind;
}

File: web-client/src/app/model/round.ts
import {Phase} from './phase';
import {Card} from './card';
import {Player} from './player';

export class Round {
  id: number;
  cards: Card[];
  playersInRound: Player[];
  currentPhase: Phase;
  button: number;
  isFinished: boolean;
  pot: number;

  static create() {
    return {
      id: 0,
      cards: undefined,
      playersInRound: undefined,
      currentPhase: Phase.Not_Started,
      button: 0,
      isFinished: false,
      pot: 0
    };
  }
}
